controllers/recruiters.js

```diff
@@ -0,0 +1,26 @@
+const recruiterQuery = require('../models/recruiters')
+
+/**
+ * Posts the data about the recruiter
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const addRecruiter = async (req, res) => {
+  try {
+    const result = await recruiterQuery.addRecruiterInfo(req.body, req.params.username)
+    if (!result) {
+      return res.boom.notFound('User doesn\'t exist')
+    }
+    result.message = 'Request Submission Successful!!'
+    return res.json(result)
```

Comment:
Doesn't this look good?
```JS
return res.json({
  message: 'Request Submission Successful!!',
  result
})
```

Reply:
Modified the response object as you suggested.